### PR TITLE
Intervention policy API plumbing for Simulation

### PR DESCRIPTION
**Pull request by Tom-Szendrey**
# Breaking
This is a breaking change for simulation, calibration.
This is a breaking change for past interventions (as I am renaming threshold -> timestep)

# Description
- [x] Allowing services to access the TDS' interventions as well as minor update to the README
- [x] Allowing us to utilize the new interventions for simulation and for calibration.
- [x] Optimize as a whole is being worked on by @asylves1 so I will leave this alone to avoid many merge conflicts
- [x] Update static's "threshold" to be called "timestep" which is much more accurate to its use

# Testing

** Required:
https://github.com/DARPA-ASKEM/pyciemss-service/pull/100

Utilize new interventions (**only static** at the moment) to run a simulation
image

Resolves #(issue)

### CODE REVIEW

**Review by dvince2 — APPROVED** [review]
I believe this all looks good. You'll need to regen types.ts

**Review by YohannParis — APPROVED** [review]

File: packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss.vue [hunk 1]
```diff
@@ -236,6 +236,8 @@ const modelConfig = ref<ModelConfiguration>();
 
 const modelConfigId = computed<string | undefined>(() => props.node.inputs[0]?.value?.[0]);
 const datasetId = computed<string | undefined>(() => props.node.inputs[1]?.value?.[0]);
+const policyInterventions = computed(() => props.node.inputs[2].value);
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss.vue:239]: Would be nicer if naming is a bit more consistent, you seem to be inter-swapping`policyId` and `polcyInterventions`, and `policyInterventionId`.

Comment (YohannParis) [reply packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss.vue]: Shouldn't be `policyInterventionIds` and not `policyInterventionsId`. Just to be pedantic?

Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss.vue]: I will rename all to policyInterventionId - no plural anywhere
1 ID is multiple interventions but one policy intervention

**Review by blanchco — APPROVED** [review]

**Review by mwdchang — APPROVED** [review]

**Pull request merged**